tests/unit/core/components/TestInstallButton.js

```diff
@@ -50,28 +51,34 @@ describe('<InstallButton />', () => {
       id: 'foo@personas.mozilla.org',
     };
     const root = render({ foo: 'foo', hasAddonManager: true, i18n, addon });
-    expect(root.type).toEqual('div');
-    expect(root.props.className).toEqual('InstallButton InstallButton--use-switch');
-    const switchComponent = root.props.children[0];
-    expect(switchComponent.type).toEqual(InstallSwitch);
-
-    const props = switchComponent.props;
-    expect(props.addon).toEqual(addon);
-    expect(props.className).toEqual('InstallButton-switch');
-    expect(props.foo).toEqual('foo');
-    expect(props.hasAddonManager).toBe(true);
-    expect(props.i18n).toEqual(i18n);
+    expect(root.type()).toEqual('div');
+    expect(root.hasClass('InstallButton')).toBeTruthy();
+    expect(root.hasClass('InstallButton--use-switch')).toBeTruthy();
+
+    const switchComponent = root.childAt(0);
+    expect(switchComponent.type()).toEqual(InstallSwitch);
+
+    expect(switchComponent.prop('addon')).toEqual(addon);
+    expect(switchComponent.prop('className')).toEqual('InstallButton-switch');
+    expect(switchComponent.prop('foo')).toEqual('foo');
+    expect(switchComponent.prop('hasAddonManager')).toBe(true);
+    expect(switchComponent.prop('i18n')).toEqual(i18n);
   });
 
   it('renders a theme button when mozAddonManager is not available', () => {
     const addon = { ...fakeAddon, type: ADDON_TYPE_THEME };
     const root = render({ hasAddonManager: false, addon });
-    expect(root.type).toEqual('div');
-    expect(root.props.className).toEqual('InstallButton InstallButton--use-button');
-    const buttonComponent = root.props.children[1];
-    expect(buttonComponent.type).toEqual(Button);
-    expect(buttonComponent.props.children).toEqual('Install Theme');
-    expect(buttonComponent.props['data-browsertheme']).toEqual(JSON.stringify(themePreview.getThemeData(addon)));
+
+    expect(root.type()).toEqual('div');
+    expect(root.hasClass('InstallButton')).toBeTruthy();
+    expect(root.hasClass('InstallButton--use-button')).toBeTruthy();
+
+    const button = root.childAt(1);
+
+    expect(button.type()).toEqual(Button);
+    expect(button.contains('Install Theme')).toBeTruthy();
```

Comment:
This should be [`expect(button).toIncludeText('Install Theme')`](https://github.com/blainekasten/enzyme-matchers#toincludetexttextstring).